hooks/hooks.js

```diff
@@ -0,0 +1,15 @@
+import React, {useState} from 'react'
```

Comment:
Think this was a good call to create a separate doc for hooks to keep the component files less busy and also now this can be reused in the future (e.g. if we implement an upvote/downvote system for disputed rules)